### PR TITLE
Implement simplify and detail for FixedPrecisionTensor

### LINKED ISSUE #2304: Serialize / Deserialize FixedPrecisionTensors
**Is your feature request related to a problem? Please describe.**

I want to be able to serialize/deserialize FixedPrecisionTensors.

Tests should be implemented.

### PR DESCRIPTION
**Pull request by mari-linhares**
Fixes #2304

### CODE REVIEW

File: syft/frameworks/torch/tensors/interpreters/precision.py [hunk 1]
```diff
@@ -515,3 +517,60 @@ def get(self):
...
+
+        Returns:
+            tuple: a tuple holding the unique attributes of the fixed precision tensor.
+        """
+        chain = None
+        if hasattr(tensor, "child"):
+            chain = syft.serde._simplify(tensor.child)
+
+        return (
+            syft.serde._simplify(tensor.id),
```
Comment (robert-wagner) [inline syft/frameworks/torch/tensors/interpreters/precision.py]: do we need to serialize these? they are strings/ints which msgpack handles fine

File: syft/frameworks/torch/tensors/interpreters/precision.py [hunk 2]
```diff
@@ -515,3 +517,60 @@ def get(self):
...
+            syft.serde._simplify(tensor.field),
+            syft.serde._simplify(tensor.base),
+            syft.serde._simplify(tensor.precision_fractional),
+            syft.serde._simplify(tensor.kappa),
+            syft.serde._simplify(tensor.tags),
```
Comment (robert-wagner) [inline syft/frameworks/torch/tensors/interpreters/precision.py]: This is the one exception to the above comment

File: syft/frameworks/torch/tensors/interpreters/precision.py (hunk 1 above)
Comment (mari-linhares) [reply syft/frameworks/torch/tensors/interpreters/precision.py]: Oops, indeed.

For ints I'm sure we don't need to simplify. But for strings we're using encoding something:

```
def _simplify_str(obj: str) -> tuple:
    return (obj.encode("utf-8"),)

```

Comment (mari-linhares) [reply syft/frameworks/torch/tensors/interpreters/precision.py]: Given that said, I'll keep simplify for attributes that can be strings (potentially) if that's okay.

Comment (robert-wagner) [reply syft/frameworks/torch/tensors/interpreters/precision.py]: sounds good to me

**Pull request merged**